src/reference/asciidoc/index.adoc

```diff
@@ -1,8 +1,17 @@
 [[spring-amqp-reference]]
-
 = Spring AMQP
+:toc: left
+:toclevels: 4
+:numbered:
+:icons: font
+:hide-uri-scheme:
+Mark Pollack; Mark Fisher; Oleg Zhurakousky; Dave Syer; Gary Russell; Gunnar Hillert; Artem Bilan; Stéphane Nicoll; Arnaud Cogoluègnes; Jay Bryant
+
+(C) 2010 - 2019 by Pivotal Software, Inc.
+
+Copies of this document may be made for your own use and for distribution to others, provided that you do not charge any fee for such copies and further provided that each copy contains this Copyright Notice, whether distributed in print or electronically.
 
-:toc:
+Version {project-version}
```

Comment:
I think we don't need to say `version` here. Plus it would be great to have it in **bold** style.
Also it looks like currently we have it before the Copyright.

Also a new PDF view doesn't show the version, neither Copyright.

I might agree though do not show `Authors` word any more:

![image](https://user-images.githubusercontent.com/1071590/51763740-3dca4380-20a1-11e9-98d5-c5f1a33ce016.png)


@garyrussell , WDYT?

Reply:
Looks nice - thanks Jay!

> think we don't need to say version here

I presume you mean just the word "version" (like with the current docs).

I honestly don't mind either way, but I think having the word is probably better than not.

Reply:
OK! 

Then it still has to be moved before Copyright and be bold anyway! 😄 

Although the problem with PDF remains...